components/meter/lang/en.json

```diff
@@ -0,0 +1,10 @@
+{
+  "fraction": {
+    "translation": "{x} / {y}",
+    "context": "A fraction of tasks that have been done. {x} is the numerator and {y} the denominator."
+  },
+  "progressIndicator": {
+    "translation": "Progress Indicator",
+    "context": "An image that represents a meter such that it shows the amount of progress on a measurable task."
```

Comment:
Crazy! Didn't know the serge stuff supported context. Is it now a best practice to use it?

Reply:
Well, the announcement will hopefully come out soon about it being best practice. I worked with Jeff over inspiration to start adding context to the translations. I've converted 2 repos in BHC. New standard that I am trying to push.